docs/src/man/getting_started.md

```diff
@@ -218,6 +218,41 @@ SQLite.load!(df, db, "dataframe_table")
 df = df |> @map({a=_.a + 1, _.b}) |> DataFrame
 ```
 
+A particular common case of a collection that supports the
+[Tables.jl](https://github.com/JuliaData/Tables.jl) interface is
+a vector of `NamedTuple`s:
+```
+julia> v = [(a=1,b=2), (a=3,b=4)]
+2-element Array{NamedTuple{(:a, :b),Tuple{Int64,Int64}},1}:
+ (a = 1, b = 2)
+ (a = 3, b = 4)
+
+julia> df = DataFrame(v)
+2×2 DataFrame
+│ Row │ a     │ b     │
+│     │ Int64 │ Int64 │
+├─────┼───────┼───────┤
+│ 1   │ 1     │ 2     │
+│ 2   │ 3     │ 4     │
+```
+You can also easily convert a data frame back to a vector of `NamedTuples`.
+Here we present two ways that this can be achieved, one using `Tables.rowtable`
```

Comment:
```suggestion
Here we present two ways that this can be achieved, one using the `Tables.rowtable`
```
But I'd just stop after "achieved", as the test of the sentence merely repeats the code. :-)

Finally, isn't the first way much faster? In that case maybe not recommend the second one?

Reply:
The first way is only 50% faster (which was a surprise to me), but it is consistently faster, so I will stick to your recommendation and drop the other way (I wanted to show the user how to use pure DataFrames.jl solution, but you are right that it is better to show good solutions).